nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/BatchManagerComponent.java

```diff
@@ -148,11 +178,20 @@ public void addStream(String batchId, String index, InputStream is, int chunkCou
     @Override
     public void addBlob(String batchId, String index, Blob blob, int chunkCount, int chunkIndex, String name,
             String mime, long fileSize) throws IOException {
+        addBlob(null, batchId, index, blob, chunkCount, chunkIndex, name, mime, fileSize);
+    }
+
+    public void addBlob(String provider, String batchId, String index, Blob blob, int chunkCount, int chunkIndex, String name,
```

Comment:
Do you want to expose this in the interface?

Reply:
I'm not certain about this. I was expecting to brainstorm this before going to the interface